Pipfile

```diff
@@ -17,6 +17,10 @@ gunicorn = "==19.9.0"
 pytz = "==2018.5"
 social-auth-app-django = "==2.1.0"
 whitenoise = "==3.3.1"
+faker = "*"
```

Comment:
Are you able to remove these remove the Pipfile? They are already in this file and are version pinned. Will throw a key error on deployment otherwise.